### DISCUSSION TITLE
Snap package not work on Kubuntu 21.04

### DISCUSSION THREAD

**Discussion by deslancer** [massCodeIO/massCode#86] [Q&A]
![Screenshot_20220526_111626](https://user-images.githubusercontent.com/77125713/170447909-1dac6a02-1b11-47c8-8749-61ecac467f42.png)

Can you compile .deb or AppImage version ?

**Comment by antonreshetov** [comment]
I started the discussion here - https://github.com/massCodeIO/massCode/discussions/87
